android/java/res/values-night/brave_colors.xml

```diff
@@ -251,4 +251,8 @@
     <color name="vpn_timer_bg_color">#0D0F14</color>
     <color name="vpn_timer_text_color">#F6F7F8</color>
     <color name="vpn_timer_icon_color">#A1ABBA</color>
+
+    <color name="text_primary">#F3F5F7</color>
```

Comment:
maybe a comment of what colours for or name of colours different like others we have for vpn `vpn...`

Reply:
i am using the names from the nala tokens. and moving forward i am planning to replace existing colors with nala tokens to keep it uniform across different platforms